vault-granular-access-control/step-admin-1-answer.md

```diff
@@ -0,0 +1,83 @@
+## Enact the policy
+
+The app requires the `read` capability for the path `external-apis/data/socials/twitter`.
+
+```hcl
+path "external-apis/data/socials/twitter" {
+  capabilities = [ "create", "read", "update", "delete" ]
+}
+
+path "external-apis/data/socials/+" {
+  capabilities = [ "create", "read", "update", "delete" ]
+}
+
+path "external-apis/undelete/socials/+" {
+  capabilities = [ "update" ]
+}
+```
+
+Append the policy definition to the local policy file.
+
+```shell
+echo "
+path \"external-apis/data/socials/twitter\" {
+  capabilities = [ \"create\", \"read\", \"update\", \"delete\" ]
+}
+
+path \"external-apis/data/socials/+\" {
+  capabilities = [ \"create\", \"read\", \"update\", \"delete\" ]
+}
+
+path \"external-apis/undelete/socials/+\" {
+  capabilities = [ \"update\" ]
+}
+" >> admins-policy.hcl
+```{{execute}}
```

Comment:
If we use the `editor-terminal` layout, you can do...

```suggestion
Open the `admins-policy.hcl`{{open}} and append the following policies.

<pre class="file" data-filename="admins-policy.hcl" data-target="append">
path "external-apis/data/socials/twitter" {
  capabilities = [ "create", "read", "update", "delete" ]
}
path "external-apis/data/socials/+" {
  capabilities = [ "create", "read", "update", "delete" ]
}
path "external-apis/undelete/socials/+" {
  capabilities = [ "update" ]
}
</pre>
```

That's the trick I use in the [Raft HA Storage](https://www.katacoda.com/hashicorp/scenarios/vault-raft-ha) tutorial.